plugin/pulpcore/plugin/stages/declarative_version.py

```diff
@@ -107,6 +116,10 @@ def create(self):
                     QueryExistingContentUnits(), ContentUnitSaver(),
                     ContentUnitAssociation(new_version)
                 ]
+                for elem in self.inject_before:
+                    stages.insert(stages.index(elem[0]), elem[1])
```

Comment:
@gmbnomis you're right about this. The idea was that it would specify the class object, not the instance. Thank you for pointing this out.

@vdusek the finding of the index to insert before/after needs to be searched using isinstance() and not for a specific instance of that class type, e.g. `stages.index(...)`. Does this make sense to you also?